mission-control-client/src/components/layout/Layout.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Nav from "./Nav";
+import Footer from "./Footer";
+
+const Layout = ({ children }) => {
+  return (
+    <>
+      <Nav />
+      <></>
```

Comment:
What is this?

Reply:
Inside of the fragments you should pull in the children prop so that the Nav and the Footer wraps anything that comes between it. I have the Layout component in App.js wrapping the entire app, so the full app should fall inside of the Layout.

In other words this is set up so that the Nav and Footer wraps everything